front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PluginInfoController.java

```diff
@@ -56,6 +62,7 @@ PluginInfo upsert(@RequestBody PluginInfo pluginInfo) {
     return pluginInfoService.upsert(pluginInfo);
   }
 
+  @PreAuthorize("@pluginInfoController.admin")
```

Comment:
I think you should be able to `@PreAuthorize("@fiatPermissionEvaluator.isAdmin()")` (we are doing that elsewhere)

Reply:
In this case, I don't need to inject it into the controller then, right?

Reply:
nope, it should be all wired up for security annotation processing